ministryofjustice/correspondence_tool_staff
# CT-3232 Create the complaint case from offender SAR

## Description

This PR is to
1 - Add the feature of raising the complaint from Offender SAR case
2 - Add a stamp message when a offender SAR links to a complaint case
3 - Added some initial states for complaint workflow

## Self-review checklist


### Screenshots


### Related JIRA tickets

https://dsdmoj.atlassian.net/secure/RapidBoard.jspa?rapidView=25&modal=detail&selectedIssue=CT-3232
https://dsdmoj.atlassian.net/secure/RapidBoard.jspa?rapidView=25&modal=detail&selectedIssue=CT-3259
### Deployment


### Manual testing instructions

1 - Login as Branson user
2 - Choose.a closed offender sar case
3 - should see a button named as 'Start complaint'
4 - Follow the steps to create a complaint case
5 - once the steps is done, a stamp message should appear in the history of linked offender sar case
5 - The initial state of the complaint case is "To be assessed"
